### PR TITLE
Added RTL support

### PR DESCRIPTION
**Pull request by w8r**
| Before | After |
| --- | --- |
| Screenshot 2023-12-06 at 11 05 43 | Screenshot 2023-12-06 at 11 06 01 |

### CODE REVIEW

File: src/components/Ogma.tsx [hunk 1]
```diff
@@ -15,10 +15,12 @@ import {
   EdgeList,
 } from "@linkurious/ogma-linkurious-parser";
 import { IOgmaConfig, PopulatedVisualization } from "@linkurious/rest-client";
-import { useAnnotationsContext, useAppContext } from "../context";
-import { getBoundingBox, mergeBounds } from "../utils";
+import { useAppContext } from "../context";
+import { getBoundingBox } from "../utils";
 import { GraphSchema } from "../api";
-import { getAnnotationsBounds } from "@linkurious/annotations-control";
+import rtl from "@mapbox/mapbox-gl-rtl-text";
```
Comment (davidrapin) [inline src/components/Ogma.tsx:21]: i cannot find the module in the package.json dependencies, is it missing?

File: src/types/index.d.ts [hunk 2]
```diff
@@ -4,3 +4,11 @@ declare module "jspdf" {
     getPageHeight: () => number;
   }
 }
+
+declare module "@mapbox/mapbox-gl-rtl-text" {
```
Comment (davidrapin) [inline src/types/index.d.ts:8]: why do we need to declare this?
doesn't the module have its own TS declaration file?

**Review by davidrapin — COMMENTED** [review]
Added couple of questions.

File: src/types/index.d.ts (hunk 2 above)
Comment (w8r) [reply src/types/index.d.ts:8]: mapbox is still posessed by Flow, they have no declaration file

**Review by davidrapin — APPROVED** [review]

**Pull request merged**